specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -123,3 +123,13 @@ swagger-to-sdk:
   - repo: azure-sdk-for-net
   - repo: azure-sdk-for-python
```

Comment:
SDK aspect, if you have Java in target release, please add a line
```
  - repo: azure-sdk-for-java
```
here.

Reply:
We can make this change on the main feature branch PR - this PR is just for the FHIR feature